### DISCUSSION TITLE
改了reality_vision_inbounds的dest之后，reality_grpc的就会无法使用，请问还需要改哪里？

### DISCUSSION THREAD

**Discussion by shijia7** [mack-a/v2ray-agent#680] [Q&A]
如题，改了reality_vision_inbounds的dest之后，reality_grpc的就无法使用，请问还需要改哪里？是代码写死了吗？

**Comment by mack-a** [comment]
还需要改grpc那个文件

**Reply by shijia7** [reply]
那为何默认生成的grpc没有关于dest和任何reality的配置却可以使用哪？ 纯好奇

**Reply by mack-a** [reply]
> 那为何默认生成的grpc没有关于dest和任何reality的配置却可以使用哪？ 纯好奇

VLESS reality vision->回落->VLESS reality grpc
